src/modules/feed/components/GuidelineInput/GuidelineInput.tsx

```diff
@@ -150,7 +151,6 @@ class GuidelineInput extends Component<Props, State> {
                     }),
                   }}
                   isClearable
-                  autoFocus
```

Comment:
Removing the autofocus will reintroduce the issue where the input loses focus when changing the value using only the keyboard. The root cause is the fact that the whole component rerenders when the value is updated, I think. Enabling autofocus was/is an easy workaround, but there might be a better solution.

Reply:
Should be picked up later in refactoring phase